src/containers/EmployeeList/styles.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '@scss/mixins';
+
+.mainBox {
+  position: relative;
+  box-shadow: 0 0.1rem 0.25rem rgb(0 0 0 / 8%);
+  min-width: 0;
+  word-wrap: break-word;
+  border: 1 solid rgba(222, 226, 230, 70%);
+  border-radius: 0.25rem;
+}
+
+.contentTitle {
+  margin: 0;
+}
+
+.boxHeader {
+  padding: 1.3rem;
+  margin-bottom: 0;
+  background-color: transparent;
+  border-bottom: 1px solid rgba(222, 226, 230, 70%);
+  @media (min-width: 710px) {
+    padding: 1.3rem 1.8rem 0;
+  }
+}
+
+.containerBoxHeader {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(0 * -1);
+  justify-content: space-between;
```

Comment:
Flex properties can be shortened to 

```scss
@include flex-common($justify: space-between, $wrap: wrap)
```

Always think of `flex-common` when working with flex that have more than 1 properties.